api/index.js

```diff
@@ -97,92 +185,99 @@ app.get('/proof/:trackingId', async (req, res) => {
     var decrypt = req.sanitizeQuery('decrypt').toBoolean();
 
     var path = iberaServicesEndpoint + `/api/proof/${trackingId}?userId=${userId}&decrypt=${decrypt}`;
-    
+
     try {
       var result = await request.get(path, {
```

Comment:
this just makes the code look less readable IMO
as a guideline, whatever can fit into one line- is ok...
breaking lines make sense when there is more than 1 param in an object, or when the parameter name/value is too long...
please use one line... h&e (here & everywhere)

Reply:
done h&e..